### DISCUSSION TITLE
Job stuck for 20 minutes

### DISCUSSION THREAD

**Discussion by CorentinAndre** [renovatebot/renovate#31370] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I was trying to setup renovate on my repository. One of the renovate job has been running for more than 20 minutes and did not finished. I can't stop it manually from the dashboard. I've tried to nuke renovate, without luck.
Here is the app (the repository is private) https://developer.mend.io/github/shinetools/app
And here is the stuck job https://developer.mend.io/github/shinetools/app/-/job/86960402-cdae-4098-bd99-18c59ac1afeb

How can I trigger the flow again in order to create a new renovate PR ? I've renamed the old PR to old renovate like it's specified in the documentation.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by CorentinAndre** [comment]
And while investigating network requests made by the website, I found out the following errorMessage
```

errorMessage
: 
"{\"fail\":\"Unknown error, please contact support\",\"log_ctx\":\"769ae7ecc7b44f77abe6ac17687a9903\"}
```

**Comment by rarkins** [comment]
The job in question already finished a few minutes ago and a new one has started
